### PR TITLE
Tennis  - Statistic graphic and minor fixes

### PR DESCRIPTION
**Pull request by peskdale**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,4 +2,5 @@
 .DS_Store
 .vagrant
 node_modules
-Node.js.lnk
\ No newline at end of file
+Node.js.lnk
+package-lock.json
```
Comment (benjaminkay93) [inline .gitignore]: package-lock should be added, it is now actually added to master to lock dependancies down, given we only ever touch this repo once a year, we should lock it to minimise dependancy failures

**Review by benjaminkay93 — DISMISSED** [review]
apart from the .gitignore, im happy with this

**Comment by benjaminkay93** [comment]
and theres a few conflicts now with master which i presume is just because of lines moving around?

File: public/index.html [hunk 2]
```diff
@@ -369,6 +369,63 @@ <h1 style="background-color: {{grid.headingcolor}} !important;">{{ grid.header }
...
+			<div class="tennis tennisStats center scale-fade" ng-show="tennisOptions.showStats">
+			   	    <div class="header">
+					    <h1>Match Statistics</h1>
+				    </div>
+					<div class="info" style="width: 100%">{{tennisOptions.matchName}} - After {{tennisScore.sets1.length}} of {{tennisOptions.maxSets}} sets</div>
+
+					<div class="row">
+						<div class="stats names">
+						    <span class="set" ng-repeat="i in tennisScore.sets1 track by $index">{{tennisScore.sets1[$index] + "-" + tennisScore.sets2[$index]}}</span>
+							<span style="float:left;">{{tennisOptions.player1}}</span>
```
Comment (stephenrob) [inline public/index.html]: Inline floats seem a bit odd given we are using classes for left and right alignment in other places

**Review by stephenrob — CHANGES_REQUESTED** [review]
Style classes over inline styles is preferred unless there's a reason for it in this case?

File: public/index.html (hunk 2 above)
Comment (peskdale) [reply public/index.html]: Fixed.

File: .gitignore (hunk 1 above)
Comment (peskdale) [reply .gitignore]: Fixed.

**Review by stephenrob — APPROVED** [review]

**Pull request merged**